android/app/src/main/res/values/strings.xml

```diff
@@ -4,5 +4,5 @@
     <string name="facebook_client_token">b42041fa82ca0989d9897db43053b85a</string>
     <string name="appCenterCrashes_whenToSendCrashes" moduleConfig="true" translatable="false">ALWAYS_SEND</string>
     <string name="appCenterAnalytics_whenToEnableAnalytics" moduleConfig="true" translatable="false">ALWAYS_SEND</string>
-    <string moduleConfig="true" name="CodePushDeploymentKey">@string/CODE_PUSH_DEPLOYMENT_KEY</string>
+    <string moduleConfig="true" name="CodePushDeploymentKey">b124</string>
```

Comment:
@emensah77 did you do this?

Reply:
In order to be able to run it locally. You can change it back